backend/sozisel/lib/sozisel/model/events.ex

```diff
@@ -35,6 +40,13 @@ defmodule Sozisel.Model.Events do
     Repo.delete(event)
   end
 
+  def clone_event(%Event{} = event) do
+    event
+    |> Utils.from_deep_struct()
+    |> Map.merge(%{id: nil})
+    |> create_event()
```

Comment:
Are you sure that this is right? WHen you clone the event I guess you should change the owner and the template that the event is refering to, right?

Reply:
Or am I missing something?